src/test/unit/StrategyManagerUnit.t.sol

```diff
@@ -163,20 +121,249 @@ contract StrategyManagerUnitTests is Test, Utils {
+        uint256 nonceAfter = strategyManager.nonces(staker);
+
+        if (expiry >= block.timestamp && expectedRevertMessageIsempty) {
+            require(sharesAfter == sharesBefore + shares, "sharesAfter != sharesBefore + shares");
+            require(nonceAfter == nonceBefore + 1, "nonceAfter != nonceBefore + 1");
+        }
+        return signature;
+    }
+
+    /**
+     * @notice internal function to help check if a strategy is part of list of deposited strategies for a staker
+     * Used to check if removed correctly after withdrawing all shares for a given strategy
+     */
+    function _isDepositedStrategy(address staker, IStrategy strategy) internal view returns (bool) {
+        uint256 stakerStrategyListLength = strategyManager.stakerStrategyListLength(staker);
+        for (uint256 i = 0; i < stakerStrategyListLength; ++i) {
+            if (strategyManager.stakerStrategyList(staker, i) == strategy) {
+                return true;
+            }
+        }
+        return false;
+    }
+
+    /**
+     * @notice Deploys numberOfStrategiesToAdd new strategies and adds them to the whitelist
+     */
+    function _addStrategiesToWhitelist(uint8 numberOfStrategiesToAdd) internal returns (IStrategy[] memory) {
+        IStrategy[] memory strategyArray = new IStrategy[](numberOfStrategiesToAdd);
+        // loop that deploys a new strategy and adds it to the array
+        for (uint256 i = 0; i < numberOfStrategiesToAdd; ++i) {
+            IStrategy _strategy = _deployNewStrategy(dummyToken, strategyManager, pauserRegistry, dummyAdmin);
+            strategyArray[i] = _strategy;
+            require(!strategyManager.strategyIsWhitelistedForDeposit(_strategy), "strategy improperly whitelisted?");
+        }
+
+        cheats.startPrank(strategyManager.strategyWhitelister());
+        for (uint256 i = 0; i < numberOfStrategiesToAdd; ++i) {
+            cheats.expectEmit(true, true, true, true, address(strategyManager));
+            emit StrategyAddedToDepositWhitelist(strategyArray[i]);
+        }
+        strategyManager.addStrategiesToDepositWhitelist(strategyArray);
+        cheats.stopPrank();
+
+        for (uint256 i = 0; i < numberOfStrategiesToAdd; ++i) {
+            require(
+                strategyManager.strategyIsWhitelistedForDeposit(strategyArray[i]),
+                "strategy not properly whitelisted"
+            );
+        }
 
-        // excude the zero address, the proxyAdmin and the eigenPodManagerMock from fuzzed inputs
-        addressIsExcludedFromFuzzedInputs[address(0)] = true;
-        addressIsExcludedFromFuzzedInputs[address(proxyAdmin)] = true;
-        addressIsExcludedFromFuzzedInputs[address(eigenPodManagerMock)] = true;
+        return strategyArray;
     }
+}
 
-    function testCannotReinitialize() public {
+contract StrategyManagerUnitTests_initialize is StrategyManagerUnitTests {
+    function test_CannotReinitialize() public {
```

Comment:
Nit: test for verifying that storage was properly initialized, as too the pause, owner, and whitelister

Reply:
added https://github.com/Layr-Labs/eigenlayer-contracts/pull/293/commits/f69019cbc2231fe2635341ebcc15f9992ea305c2